src/utilities.js

```diff
@@ -122,6 +122,16 @@ export function getResults(searchType, obj) {
 
   return request(options)
   .then((results) => {
+
+    // If this fails, you've hit the rate limit or Google has changed something
+    try {
+      JSON.parse(results.slice(4));
+    } catch (e) {
+      // Throw the JSON error - This should perhaps throw an object with the
+      // body of the request so the actual issue can be determined
+      throw e;
+    }
+
     const parsedResults = JSON.parse(results.slice(4)).widgets;
```

Comment:
Yea, `const` is block scoped.  You'll need to move `parsedResults` outside of the `try catch` and initialize it with `let` instead of `const`.  Then inside of the `try catch` you can reassign `parsedResults = JSON.parse(...`.  Or if you move this to a function, you can return `parsedResults` from within the `try` block if it completes successfully, or throw an error within the `catch` that contains the body of the request/response